common/test/model/ContentTest.scala

```diff
@@ -141,6 +147,59 @@ class ContentTest extends FlatSpec with Matchers with GuiceOneAppPerSuite with i
   }
 
 
+  val dateBeforeCutoff = Some(jodaToJavaInstant(new DateTime("2017-07-02T12:00:00.000Z")).atOffset(ZoneOffset.UTC).toCapiDateTime)
+  val dateAfterCutoff = Some(jodaToJavaInstant(new DateTime("2017-07-04T12:00:00.000Z")).atOffset(ZoneOffset.UTC).toCapiDateTime)
+  val edition: Edition = mock[Edition]
+  val sensitiveContentFields = Some(ContentFields(sensitive = Some(true)))
+  val nonSensitiveContentFields = Some(ContentFields(sensitive = Some(true)))
+
+
+  when(edition.id) thenReturn "GB"
+
+  "ContentJavascriptConfig" should "set shouldHideReaderRevenue to true for a sensitive article, published before the cutoff date, no RR flag" in {
```

Comment:
spacing's a bit weird, looks like `ContentJavascriptConfig` is a variable name - maybe "Content JavaScript config"